Makefile

```diff
@@ -91,7 +91,7 @@ user:
 	. ./$(USER_SOURCE) && make all
 
 .PHONY: all
-all: setup templates lint fixrights test
+all: setup templates lint fixrights testunit
```

Comment:
jenkins doesn't use all as a target. 

'build' is using cleanall setup template, 
'lint' is using lint
'fix rights' is using fixrights
and tests were already divided between functional and integration in test phase.

This would remove integration tests from the deploy steps. I do not know it this is the intended purpose, but if that was not the case, I would not change this.

Reply:
well, Jenkins is in stale right now, so I don't really want to fix it 😛

Reply:
I've disabled integration tests on Jenkins, and added them after every `make all` performed while deploying. This should do the trick while waiting for CI to migrate to CodeBuild

Reply:
Works for me.